docs/advanced/custom_readers.md

```diff
@@ -0,0 +1,155 @@
+
+rio-tiler provides multiple base classes from which it derives its main readers: [COGReader](/readers/#cogreader) and [STACReader](/readers/#stacreader). The base classes can be used by users to built custom readers (see [rio-tiler-pds](https://github.com/cogeotiff/rio-tiler-pds)).
+
+## Abstract Base Classes
+
+### ø BaseReader
```

Comment:
Why `ø`? It's probably ok just to have h3 headers as the classes.

Reply:
I found the (right side) menu a bit hard to read (when mkdocs is rendered). I'm happy to remove this!